server/routes/aws/cloudwatch_tasks.py

```
@@ -0,0 +1,545 @@
+            alert_severity=severity,
+            correlation_result=correlation_result,
+            alert_metadata=alert_metadata, raw_payload=payload,
+            org_id=org_id,
+        )
+        cursor.execute("RELEASE SAVEPOINT sp_handle_correlated")
+        conn.commit()
+        logger.info(
+            "[CLOUDWATCH][ALARM] Correlated alarm '%s' to incident %s",
+            alarm_name, correlation_result.incident_id,
+        )
+    except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)